### PR TITLE
[#40498] Update OpenID documentation (add Google)

### PR DESCRIPTION
**Pull request by ghost**
[#40498] Update OpenID documentation (add Google)

https://community.openproject.org/work_packages/40498

### CODE REVIEW

File: docs/system-admin-guide/authentication/openid-providers/README.md [hunk 1]
```diff
@@ -24,7 +24,88 @@ You can configure the following options.
...
+## Google Workspace
+
+
+
+### Step 1: Create the OAuth consent screen
+
+1. Navigate to your GCP console.  (https://console.cloud.google.com/)
+2. Go to **APIs & Services** > OAuth consent screen.
+
+![image-20220215095201951](/home/ant0n/.config/Typora/typora-user-images/image-20220215095201951.png)
```
Comment (oliverguenther) [inline docs/system-admin-guide/authentication/openid-providers/README.md]: The paths to the images are not correct, they need to be relative to the README.md file, for example `./images/image-20220215095201951` if you put them in a subfolder `images`

You can configure Typora to move the images relative to the file

image

File: docs/system-admin-guide/authentication/openid-providers/README.md [hunk 2]
```diff
@@ -24,7 +24,88 @@ You can configure the following options.
...
+
+3. Create a new project and a new app or edit an existing project and an existing app, setting the following fields (shall be Internal):
+   1. **App name** (e.g. EXAMPLE.COM SSO)
+   2. **User support email** (e.g. user-support@example.com)
+   3. **App domains** (at minimum, you must provide the Application home page - e.g. https://example.openproject.com)
+   4. **Authorized domains** (e.g. openproject.com)
+   5. **Developer Contact information** (e.g.  developer@example.com)
+   6. Click **SAVE AND CONTINUE** to proceed.
+
+![image-20220215095953811](/home/ant0n/.config/Typora/typora-user-images/image-20220215095953811.png)
```
Comment (oliverguenther) [inline docs/system-admin-guide/authentication/openid-providers/README.md]: Same here

File: docs/system-admin-guide/authentication/openid-providers/README.md [hunk 3]
```diff
@@ -24,7 +24,88 @@ You can configure the following options.
...
+4. **Scopes** - Press **SAVE AND CONTINUE**
+5. **Summary** - Press **SAVE AND CONTINUE**
+
+
+
+### Step 2: Create the OAuth Client
+
+1. Under **APIs & Services**, go to **Credentials**.
+
+![image-20220215102049207](/home/ant0n/.config/Typora/typora-user-images/image-20220215102049207.png)
```
Comment (oliverguenther) [inline docs/system-admin-guide/authentication/openid-providers/README.md]: Same here

File: docs/system-admin-guide/authentication/openid-providers/README.md [hunk 4]
```diff
@@ -24,7 +24,88 @@ You can configure the following options.
...
+
+   1. When prompted for your **Application type**, choose **Web Application**.
+
+   2. Provide a **Name** for your application. (e.g. example-openproject-com)
+
+   3. Under Authorized redirect URIs, click **Add URI**, and provide your URI (e.g. [example.openproject.com]/auth/google/callback).
+
+   4. Click **CREATE** or **SAVE** .
+
+![image-20220215102234492](/home/ant0n/.config/Typora/typora-user-images/image-20220215102234492.png)
```
Comment (oliverguenther) [inline docs/system-admin-guide/authentication/openid-providers/README.md]: Same here

File: docs/system-admin-guide/authentication/openid-providers/README.md [hunk 5]
```diff
@@ -24,7 +24,88 @@ You can configure the following options.
...
+
+After pressing **CREATE** you will get a pop-up window like the following
+
+- Note **Client ID**
+- Note **Client Secret**
+
+![image-20220215102658395](/home/ant0n/.config/Typora/typora-user-images/image-20220215102658395.png)
```
Comment (oliverguenther) [inline docs/system-admin-guide/authentication/openid-providers/README.md]: Same here

File: docs/system-admin-guide/authentication/openid-providers/README.md [hunk 6]
```diff
@@ -24,7 +24,88 @@ You can configure the following options.
...
+
+1. Login as Open Project Administrator
+2. navigate to -> *Administration* -> *Authentication* and choose -> *OpenID providers*.
+   1. **Name** Choose Google
+   2. **Display Name** (e.g. **EXAMPLE.COM SSO**)
+   3. **Identifier** (**Client ID** from step 2)
+   4. **Secret** (**Client Secret** from step 2)
+3. Press **Create**
+
+![image-20220215103036615](/home/ant0n/.config/Typora/typora-user-images/image-20220215103036615.png)
```
Comment (oliverguenther) [inline docs/system-admin-guide/authentication/openid-providers/README.md]: Same here

File: docs/system-admin-guide/authentication/openid-providers/README.md [hunk 7]
```diff
@@ -24,7 +24,88 @@ You can configure the following options.
...
+
+4. The following green notification **Successful creation** should appear
+
+![image-20220215103344744](/home/ant0n/.config/Typora/typora-user-images/image-20220215103344744.png)
```
Comment (oliverguenther) [inline docs/system-admin-guide/authentication/openid-providers/README.md]: Same here

File: docs/system-admin-guide/authentication/openid-providers/README.md [hunk 8]
```diff
@@ -24,7 +24,88 @@ You can configure the following options.
...
+
+
+
+
+
+
```
Comment (oliverguenther) [inline docs/system-admin-guide/authentication/openid-providers/README.md:102]: Please avoid repeated newlines like that (keep them <= 2 newlines for spacing purposes)

**Review by oliverguenther — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
@oliverguenther I corrected the requested changes, comitted, pushed origin, what next?

**Review by oliverguenther — APPROVED** [review]

**Pull request merged**